vm/src/import.rs

```diff
@@ -67,7 +67,7 @@ pub fn import_file(
 
 fn find_source(vm: &VirtualMachine, current_path: PathBuf, name: &str) -> Result<PathBuf, String> {
     let sys_path = vm.get_attribute(vm.sys_module.clone(), "path").unwrap();
-    let mut paths: Vec<PathBuf> = objsequence::get_elements(&sys_path)
+    let mut paths: Vec<PathBuf> = objsequence::get_elements_list(&sys_path)
```

Comment:
The objsequence file was created with the overlapping functions of tuple and list in mind. Now that we have specific functions for tuple and list, move those two functions to the objtuple and objlist files.

Please move `get_elements_list` into the objlist.rs file and rename it to `get_elements`.
Please also move `get_elements_tuple` into the objtuple.rs file and rename it to `get_elements`.